backend/inspirehep/Brewfile

```diff
@@ -0,0 +1,27 @@
+tap "homebrew/bundle"
+tap "homebrew/cask"
+tap "homebrew/core"
+tap "int128/kubelogin"
+brew "conftest"
+brew "gcc"
+brew "kubeseal"
+brew "kustomize"
+brew "libmagic"
+brew "libpq"
+brew "openblas"
+brew "openssl@3", link: true
+brew "readline"
+brew "postgresql"
+brew "sqlite"
+brew "xz"
+brew "pre-commit"
+brew "pyenv"
+brew "pyenv-virtualenv"
+brew "pyenv-virtualenvwrapper"
+brew "python@3.8"
+brew "python@3.9"
+brew "sshuttle"
+brew "virtualenv"
+brew "yarn"
+brew "zlib"
+brew "int128/kubelogin/kubelogin"
```

Comment:
not sure if wee need k8s dependencies there @drjova?

Reply:
true, could you please @karolina-siemieniuk-morawska remove it?

Reply:
Also could you please add another section on the documentation on how to install this file?